src/Resolution.ts

```diff
@@ -346,6 +346,16 @@ export default class Resolution {
     return await method.records(domain, keys);
   }
 
+  /**
+   * @param domain domain name
+   * @returns A Promise that resolves true or false
```

Comment:
> A Promise that resolves true or false

This is pretty much obvious from the return type itself. I'd describe what is meant by being registered instead.